File: codigo/soluciones/kat/acm.py
solved = set()
attempts = dict()
ans = 0
while True:
    tokens = input().split()
    if len(tokens) == 1:
        break
    t = int(tokens[0])
    c = tokens[1]
    s = tokens[2]
    if c in solved:
        continue
    if s == 'right':
        ans += t
        solved.add(c)
    else:
        if not c in attempts:
            attempts[c] = 0
        attempts[c] += 1
for c, x in attempts.items():
    if c not in solved:
        continue
    ans += x * 20
print(len(solved), ans)

File: codigo/soluciones/kat/acm.java
import java.io.*;
import java.util.*;
class Acm {
    private void _main() {
        Scanner sc = new Scanner(new BufferedInputStream(System.in));
        HashSet<String> solved = new HashSet<>();
        HashMap<String, Integer> attempts = new HashMap<>();
        int ans = 0;
        while (true) {
            int t = sc.nextInt();
            if (t == -1) break;
            String c = sc.next();
            String s = sc.next();
            if (solved.contains(c)) continue;
            if (s.equals("right")) {
                ans += t;
                solved.add(c);
            }else {
                attempts.put(c, attempts.getOrDefault(c, 0) + 1);
            }
        }
        for (String c : attempts.keySet()) {
            if (!solved.contains(c)) continue;
            ans += attempts.get(c) * 20;
        }
        System.out.println(solved.size() + " " + ans);
        sc.close();
    }
    public static void main(String[] args) {
        (new Acm())._main();
    }
}
